symengine/cwrapper.cpp

```diff
@@ -1,4 +1,3 @@
-#include <cstdlib>
```

Comment:
`CLion` showed that this import was unused, and none of the builds failed as well.